spec/parsers/cr3_parser_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'spec_helper'
+
+describe FormatParser::CR3Parser do
```

Comment:
Could you add some negative tests as well? I.e. test that some files can't be parsed. 

Also, you can add some tests for the `likely_match?` method as well, that check for case insensitivity, as well as some negative scenarios. You can check the `aac_parser_spec.rb` file, and get inspiration from the following tests:
- "should match filenames with valid AAC extensions"
- "should not match filenames with invalid AAC extensions"

Reply:
Got some examples of negative tests? Do you mean corrupted files, for example? That's not something I've seen tested for the majority of other formats.

Reply:
Added in tests for the escape cases (when the file doesn't match the CR3 definition and when the CMT1 atom can't be found) and for the extension matching.

Reply:
Nice work! I'm happy with those tests.